packages/flutter_driver/lib/src/extension.dart

```diff
@@ -179,23 +179,26 @@ class FlutterDriverExtension {
     return constructor(finder);
   }
 
-  Future<TapResult> tap(Tap command) async {
-    prober.tap(await _waitForElement(_createFinder(command.finder)));
+  Future<TapResult> tap(Command command) async {
+    Tap tap = command;
```

Comment:
Absolutely, I agree.  I should have prefaced this PR with the caveat that this is _"NOT how I'd like to do it but I don't see any other way to make strong-mode happy, please help!"_